generators/client/templates/react/src/main/webapp/app/modules/administration/configuration/configuration.tsx.ejs

```diff
@@ -69,6 +69,9 @@ export class ConfigurationPage extends React.Component<IConfigurationPageProps,
     });
   };
 
+  getContextList = contexts => Object.values(contexts).map(v => v['beans']).reduce((acc, e) => ({ ...acc, ...e }));
```

Comment:
why not just `v => v.beans`

Reply:
It creates an error since `v` is just a regular `Object` at this point.

Reply:
what do you mean? if `v = { beans: ... }` then `v.beans` and `v['beans']` are the same

Reply:
I know, but TS don't or at least that's what vscode is telling me. I don't like doing it that way too, so I wouldn't mind a suggestion on how to improve that.

Reply:
oh, TS issue, it's due to type missing. Try `(v: any) => v.beans`

Reply:
It works, thanks :+1: